src/autolabel/data_loaders/validation.py

```diff
@@ -109,11 +109,41 @@ def validate(self, value: str):
         pass
 
 
+class MLCTaskValidate(BaseModel):
+    """Validate Multilabel Classification Task
+
+    As of now we assume that the input label_column is a string
+
+    The label column can be a delimited string or a string of list
+    """
+
+    label_column: str
+    labels_set: set  # A Multilabel Classification Task should have a unique set of labels in config
+
+    def validate(self, value: str):
+        if value.startswith("[") and value.endswith("]"):
+            try:
+                seed_labels = eval(value)
+                if not isinstance(seed_labels, list):
+                    raise
```

Comment:
trying to understand how the empty raise works - from what i read it means that it raises the last exception that was handled. Given we are not in an except clause here, what would an empty raise mean here?
I would suggest raising a ValueError just for readability

Reply:
I was just copying the other validation functions but it makes more sense to raise a `ValueError` here so I'll add that now.